rfc6962-bis.xml

```diff
@@ -555,7 +555,16 @@ SEQUENCE {
           TBSCertificate tbs_certificate;
       } CertInfo;
 
-      opaque CtExtensions&lt;0..2^16-1&gt;;
+    uint16 CtExtensionId;
```

Comment:
TLS 1.2 hello extension IDs have an associated IANA registry but they're defined as an enum rather than a uint16.  See RFC5246 Section 7.4.1.4.

We should do the same here.